lark/exceptions.py

```diff
@@ -118,47 +118,44 @@ def match_examples(self, parse_fn, examples, token_type_match_fallback=False, us
 
         return candidate[0]
 
-    def _format_terminals(self, terminals):
+    def _format_excepts(self, excepts):
         if self._terminals_by_name:
             ts = []
-            for ter in terminals:
+            for ter in excepts:
```

Comment:
what is `excepts`?

Reply:
The argument to the function

Reply:
What does the word "excepts" mean in this context?

Reply:
Typo: meant 'expected'/'expects'. English isn't my first language.

Reply:
It's not a typo if it appears 20 times :)

Reply:
7, most of which with autocomplete.